.github/workflows/tag.yaml

```diff
@@ -0,0 +1,29 @@
+name: create_tag
+on: 
+  pull_request:
+    types: [closed]
+
+permissions:
+  contents: write
+
+jobs:
+  create_tag:
+    name: "Create tag"
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'release-pr')
```

Comment:
If we only want this to be triggered when a PR titled with `chore: Prepare` and has this label is merged, then how about include the PR title as a conditional to ensure a random PR with this label doesnt trigger a new tag creation.